client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx

```diff
@@ -0,0 +1,114 @@
+import useStyles from './useStyles';
+
+import { contest, contests } from '../../interface/tempContestData';
+// mock data for now
+const submissions = [
+  { img: 'https://source.unsplash.com/random', author: 'freddy', title: 'hello' },
+  { img: 'https://source.unsplash.com/1600x900/?nature,water', author: 'Roberta', title: 'Lion Tattoo' },
+  { img: 'https://source.unsplash.com/1600x900/?lions,tigers', author: 'Carlos', title: 'My Submission' },
+  { img: 'https://source.unsplash.com/1600x900/?tattoo,plants', author: 'Wendy', title: 'Pick Me!' },
+  { img: 'https://source.unsplash.com/1600x900/?plane,eyes', author: 'Frank', title: 'Fly away' },
+  { img: 'https://source.unsplash.com/1600x900/?comet,hello', author: 'jojoe', title: 'indiana jones' },
+  { img: 'https://source.unsplash.com/1600x900/?space,fire', author: 'miranda', title: 'contest time' },
+  { img: 'https://source.unsplash.com/1600x900/?grim,reaper', author: 'Sarah', title: 'you are my friend' },
+  { img: 'https://source.unsplash.com/1600x900/?brazil,country', author: 'Paul', title: 'what is up' },
+  { img: 'https://source.unsplash.com/1600x900/?forest,fight', author: 'Milad', title: 'hello' },
+  { img: 'https://source.unsplash.com/1600x900/?tank,fighter', author: 'Carlos', title: 'hello' },
+];
+
+export default function ViewContestSubmissions(): JSX.Element {
+  const classes = useStyles();
+  const [contestCard, setContestCard] = useState<contest>(Object);
+  const [winnerIndex, setWinnerIndex] = useState(Number);
+
+  //   pulling dummy data for now
+  const handleContest = () => {
+    setContestCard(contests[0]);
+  };
+
+  useEffect(() => {
+    handleContest();
+  }, [contestCard]);
+
+  //   this is the grossest thing i've ever coded but it's just there to test functionality
+  useEffect(() => {
+    setWinnerIndex(999);
+  }, []);
+
+  const handleWinnerIndex = (index: number) => {
+    setWinnerIndex(index);
+    console.log(winnerIndex);
+  };
+
+  const renderImageList = (): JSX.Element[] => {
+    return submissions.map((tile, index) => {
+      return (
+        <GridListTile key={index}>
+          <img src={tile.img} alt={tile.title} />
+          <GridListTileBar
+            title={tile.title}
+            subtitle={<span>by: {tile.author}</span>}
+            actionIcon={
+              index === winnerIndex ? (
+                <IconButton
+                  key={index}
+                  aria-label={`info about ${tile.title}`}
+                  onClick={() => handleWinnerIndex(index)}
+                  className={classes.icon}
+                >
+                  <Chip label="Winner"></Chip>
+                </IconButton>
+              ) : (
+                <IconButton
+                  key={index}
+                  aria-label={`info about ${tile.title}`}
+                  onClick={() => handleWinnerIndex(index)}
+                  className={classes.icon}
+                >
+                  <AddCircleOutlineIcon />
```

Comment:
Maybe something like a trophy icon would make more sense? I don't know if I would intuitively assume this was the choose winner button as is

Reply:
Done